resources/instance_upstart.rb

```diff
@@ -1,130 +0,0 @@
-#
-# Cookbook:: memcached
-# resource:: instance_upstart
```

Comment:
Upstart is still a thing in RHEL 6. Folks are using this

Reply:
In the slack conversation it was discussed to remove upstart support since its a burden to maintain and everyone should be on systemd anyways. RHEL6 is in semi end of life (from 2016 already) but receives minor patches until 2020.